### ISSUE TITLE
Improve lambda logging for Offline-Snapshot / Offline-Compaction-Snapshot

### ISSUE THREAD

**Issue by mbloch1986** [shinesolutions/aem-stack-manager-cloud#29]
During the offline-snapshot / offline-compaction-snapshot it might be handy to print some additional infos to the ssm command which gets currently executed 

* Stack name
* EC2 instance
* AEM Component
* SSM Command

**Comment by mbloch1986** [comment]
Example log output for PR #30 


```
START RequestId: a2f1aaa6-b18a-11e8-b239-3d0b7a21a89a Version: $LATEST
[blochfs] Start checking Stack health ...
Try to lock DynamoDB Table deployTestSM-AemStackManagerTable ...
DynamoDB Table deployTestSM-AemStackManagerTable locked successfully.
[blochfs] Start updating ASG blochfs-aem-full-set-main-stack-AuthorDispatcherStack-UWEZUL12EO5S-AuthorDispatcherAutoScalingGroup-1N7IDYIQ158W8 to 0 instances ...
[blochfs] Finish updating ASG blochfs-aem-full-set-main-stack-AuthorDispatcherStack-UWEZUL12EO5S-AuthorDispatcherAutoScalingGroup-1N7IDYIQ158W8 to 0 instances.
[blochfs] Start entering instance [u'i-0fbc64c7647a2d4c8', u'i-04081829e6d35e02c'] into standby in ASG blochfs-aem-full-set-main-stack-AuthorDispatcherStack-UWEZUL12EO5S-AuthorDispatcherAutoScalingGroup-1N7IDYIQ158W8 ...
[blochfs] Finish entering instance [u'i-0fbc64c7647a2d4c8', u'i-04081829e6d35e02c'] into standby in ASG blochfs-aem-full-set-main-stack-AuthorDispatcherStack-UWEZUL12EO5S-AuthorDispatcherAutoScalingGroup-1N7IDYIQ158W8.
[blochfs] Start updating ASG blochfs-aem-full-set-main-stack-PublishDispatcherStack-X0LO9LX9R3CV-PublishDispatcherAutoScalingGroup-1W3ID11J40LGF to 0 instances ...
[blochfs] Finish updating ASG blochfs-aem-full-set-main-stack-PublishDispatcherStack-X0LO9LX9R3CV-PublishDispatcherAutoScalingGroup-1W3ID11J40LGF to 0 instances.
[blochfs] Start entering instance ['i-0ce2994ef2bd282e6'] into standby in ASG blochfs-aem-full-set-main-stack-PublishDispatcherStack-X0LO9LX9R3CV-PublishDispatcherAutoScalingGroup-1W3ID11J40LGF ...
[blochfs] Finish entering instance ['i-0ce2994ef2bd282e6'] into standby in ASG blochfs-aem-full-set-main-stack-PublishDispatcherStack-X0LO9LX9R3CV-PublishDispatcherAutoScalingGroup-1W3ID11J40LGF.
[blochfs/author-standby] Start sending command STOP_AUTHOR_STANDBY for instance ids:
[blochfs/author-standby] [u'i-09cf0aceca37f8f15'] ...
Start calling SSM Document deployTestSM-aem-stack-manager-main-stack-SSMCommandsStack-B4JXFJJAEMR1-ManageService-1MSVLEJ131TV5 ...
Finished calling SSM Document deployTestSM-aem-stack-manager-main-stack-SSMCommandsStack-B4JXFJJAEMR1-ManageService-1MSVLEJ131TV5.
[blochfs/author-standby] Command ID: 94713310-1b4c-4e81-a8b3-c61fe7c0ad05 ...
[blochfs/author-standby] Finished sending command STOP_AUTHOR_STANDBY
END RequestId: a2f1aaa6-b18a-11e8-b239-3d0b7a21a89a
REPORT RequestId: a2f1aaa6-b18a-11e8-b239-3d0b7a21a89a Duration: 4578.14 ms Billed Duration: 4600 ms Memory Size: 128 MB Max Memory Used: 69 MB
START RequestId: a8b30e60-b18a-11e8-b306-d7d4862bcf78 Version: $LATEST
[blochfs/author-primary] Start sending command STOP_AUTHOR_PRIMARY for instance ids:
[blochfs/author-primary] [u'i-0919e15265022a4f7'] ...
Start calling SSM Document deployTestSM-aem-stack-manager-main-stack-SSMCommandsStack-B4JXFJJAEMR1-ManageService-1MSVLEJ131TV5 ...
Finished calling SSM Document deployTestSM-aem-stack-manager-main-stack-SSMCommandsStack-B4JXFJJAEMR1-ManageService-1MSVLEJ131TV5.
[blochfs/author-primary] Command ID: 2353bdc7-47a8-4f52-acd1-4202e51c071f ...
[blochfs/author-primary] Finished sending command STOP_AUTHOR_PRIMARY
END RequestId: a8b30e60-b18a-11e8-b306-d7d4862bcf78
REPORT RequestId: a8b30e60-b18a-11e8-b306-d7d4862bcf78 Duration: 553.36 ms Billed Duration: 600 ms Memory Size: 128 MB Max Memory Used: 70 MB
START RequestId: d418d317-b18a-11e8-9467-89b6b872ecee Version: $LATEST
[blochfs/publish] Start sending command STOP_PUBLISH for instance ids:
[blochfs/publish] [u'i-06a2d10be98cc7e01'] ...
Start calling SSM Document deployTestSM-aem-stack-manager-main-stack-SSMCommandsStack-B4JXFJJAEMR1-ManageService-1MSVLEJ131TV5 ...
Finished calling SSM Document deployTestSM-aem-stack-manager-main-stack-SSMCommandsStack-B4JXFJJAEMR1-ManageService-1MSVLEJ131TV5.
[blochfs/publish] Command ID: 9a1e67d9-2f87-4538-89a3-a62c51807c31 ...
[blochfs/publish] Finished sending command STOP_PUBLISH
END RequestId: d418d317-b18a-11e8-9467-89b6b872ecee
REPORT RequestId: d418d317-b18a-11e8-9467-89b6b872ecee Duration: 1088.31 ms Billed Duration: 1100 ms Memory Size: 128 MB Max Memory Used: 70 MB
START RequestId: f750a21d-b18a-11e8-98db-b3e4fdc4042b Version: $LATEST
[blochfs/author-standby/author-primary/publish] Start sending command OFFLINE_COMPACTION for instance ids:
[blochfs/author-standby/author-primary/publish] [u'i-0919e15265022a4f7', u'i-09cf0aceca37f8f15', u'i-06a2d10be98cc7e01'] ...
Start calling SSM Document deployTestSM-aem-stack-manager-main-stack-SSMCommandsStack-B4JXFJJAEMR1-OfflineCompactionSnapshotFullset-HDTC3CRHREH3 ...
Finished calling SSM Document deployTestSM-aem-stack-manager-main-stack-SSMCommandsStack-B4JXFJJAEMR1-OfflineCompactionSnapshotFullset-HDTC3CRHREH3.
[blochfs/author-standby/author-primary/publish] Command ID: e05fbaef-31d3-46dc-a344-fdc5d98c21f3 ...
[blochfs/author-standby/author-primary/publish] Finished sending command OFFLINE_COMPACTION
END RequestId: f750a21d-b18a-11e8-98db-b3e4fdc4042b
REPORT RequestId: f750a21d-b18a-11e8-98db-b3e4fdc4042b Duration: 707.09 ms Billed Duration: 800 ms Memory Size: 128 MB Max Memory Used: 70 MB
START RequestId: 011035a4-b18b-11e8-b673-ad5e1ba046db Version: $LATEST
[blochfs/author-standby/author-primary/publish] Start sending command OFFLINE_BACKUP for instance ids:
[blochfs/author-standby/author-primary/publish] [u'i-0919e15265022a4f7', u'i-09cf0aceca37f8f15', u'i-06a2d10be98cc7e01'] ...
Start calling SSM Document deployTestSM-aem-stack-manager-main-stack-SSMCommandsStack-B4JXFJJAEMR1-OfflineSnapshotFullset-OHFNKOW8KT2P ...
Finished calling SSM Document deployTestSM-aem-stack-manager-main-stack-SSMCommandsStack-B4JXFJJAEMR1-OfflineSnapshotFullset-OHFNKOW8KT2P.
[blochfs/author-standby/author-primary/publish] Command ID: 7d80553e-1bf2-4cec-84cd-fc3b70bfaabc ...
[blochfs/author-standby/author-primary/publish] Finished sending command OFFLINE_BACKUP
END RequestId: 011035a4-b18b-11e8-b673-ad5e1ba046db
REPORT RequestId: 011035a4-b18b-11e8-b673-ad5e1ba046db Duration: 596.59 ms Billed Duration: 600 ms Memory Size: 128 MB Max Memory Used: 70 MB
START RequestId: 2b63c817-b18b-11e8-97c5-fbd21bf91424 Version: $LATEST
[blochfs/author-primary] Start sending command START_AUTHOR_PRIMARY for instance ids:
[blochfs/author-primary] [u'i-0919e15265022a4f7'] ...
Start calling SSM Document deployTestSM-aem-stack-manager-main-stack-SSMCommandsStack-B4JXFJJAEMR1-ManageService-1MSVLEJ131TV5 ...
Finished calling SSM Document deployTestSM-aem-stack-manager-main-stack-SSMCommandsStack-B4JXFJJAEMR1-ManageService-1MSVLEJ131TV5.
[blochfs/author-primary] Command ID: b369ef93-7084-4a19-964c-6c2f1405b67a ...
[blochfs/author-primary] Finished sending command START_AUTHOR_PRIMARY
END RequestId: 2b63c817-b18b-11e8-97c5-fbd21bf91424
REPORT RequestId: 2b63c817-b18b-11e8-97c5-fbd21bf91424 Duration: 1064.36 ms Billed Duration: 1100 ms Memory Size: 128 MB Max Memory Used: 70 MB
START RequestId: 365c2985-b18b-11e8-b1bb-3d5b9a8db555 Version: $LATEST
[blochfs/author-standby] Start sending command START_AUTHOR_STANDBY for instance ids:
[blochfs/author-standby] [u'i-09cf0aceca37f8f15'] ...
Start calling SSM Document deployTestSM-aem-stack-manager-main-stack-SSMCommandsStack-B4JXFJJAEMR1-ManageService-1MSVLEJ131TV5 ...
Finished calling SSM Document deployTestSM-aem-stack-manager-main-stack-SSMCommandsStack-B4JXFJJAEMR1-ManageService-1MSVLEJ131TV5.
[blochfs/author-standby] Command ID: 5fab1292-b428-4c2a-b16a-a053789d9230 ...
[blochfs/author-standby] Finished sending command START_AUTHOR_STANDBY
END RequestId: 365c2985-b18b-11e8-b1bb-3d5b9a8db555
REPORT RequestId: 365c2985-b18b-11e8-b1bb-3d5b9a8db555 Duration: 627.63 ms Billed Duration: 700 ms Memory Size: 128 MB Max Memory Used: 70 MB
START RequestId: 41072e36-b18b-11e8-ae80-7539ac00292b Version: $LATEST
[blochfs/publish] Start sending command START_PUBLISH for instance ids:
[blochfs/publish] [u'i-06a2d10be98cc7e01'] ...
Start calling SSM Document deployTestSM-aem-stack-manager-main-stack-SSMCommandsStack-B4JXFJJAEMR1-ManageService-1MSVLEJ131TV5 ...
Finished calling SSM Document deployTestSM-aem-stack-manager-main-stack-SSMCommandsStack-B4JXFJJAEMR1-ManageService-1MSVLEJ131TV5.
[blochfs/publish] Command ID: 4f407207-fcb6-4427-8fad-4e7ee9c8733a ...
[blochfs/publish] Finished sending command START_PUBLISH
END RequestId: 41072e36-b18b-11e8-ae80-7539ac00292b
REPORT RequestId: 41072e36-b18b-11e8-ae80-7539ac00292b Duration: 582.35 ms Billed Duration: 600 ms Memory Size: 128 MB Max Memory Used: 70 MB
START RequestId: 4b4edd1e-b18b-11e8-9467-89b6b872ecee Version: $LATEST
[blochfs] Start exiting instance [u'i-0fbc64c7647a2d4c8', u'i-04081829e6d35e02c'] from standby in ASG blochfs-aem-full-set-main-stack-AuthorDispatcherStack-UWEZUL12EO5S-AuthorDispatcherAutoScalingGroup-1N7IDYIQ158W8 ...
[blochfs] Finish exiting instance [u'i-0fbc64c7647a2d4c8', u'i-04081829e6d35e02c'] from standby in ASG blochfs-aem-full-set-main-stack-AuthorDispatcherStack-UWEZUL12EO5S-AuthorDispatcherAutoScalingGroup-1N7IDYIQ158W8.
[blochfs] Start updating ASG blochfs-aem-full-set-main-stack-AuthorDispatcherStack-UWEZUL12EO5S-AuthorDispatcherAutoScalingGroup-1N7IDYIQ158W8 to 2 instances ...
[blochfs] Finish updating ASG blochfs-aem-full-set-main-stack-AuthorDispatcherStack-UWEZUL12EO5S-AuthorDispatcherAutoScalingGroup-1N7IDYIQ158W8 to 2 instances.
[blochfs] Start exiting instance [u'i-0ce2994ef2bd282e6'] from standby in ASG blochfs-aem-full-set-main-stack-PublishDispatcherStack-X0LO9LX9R3CV-PublishDispatcherAutoScalingGroup-1W3ID11J40LGF ...
[blochfs] Finish exiting instance [u'i-0ce2994ef2bd282e6'] from standby in ASG blochfs-aem-full-set-main-stack-PublishDispatcherStack-X0LO9LX9R3CV-PublishDispatcherAutoScalingGroup-1W3ID11J40LGF.
[blochfs] Start updating ASG blochfs-aem-full-set-main-stack-PublishDispatcherStack-X0LO9LX9R3CV-PublishDispatcherAutoScalingGroup-1W3ID11J40LGF to 4 instances ...
[blochfs] Finish updating ASG blochfs-aem-full-set-main-stack-PublishDispatcherStack-X0LO9LX9R3CV-PublishDispatcherAutoScalingGroup-1W3ID11J40LGF to 4 instances.
[blochfs/publish] Start sending command COMPACT_REMAINING_PUBLISHERS / PUBLISH_READY for instance ids:
[blochfs/publish] [u'i-06a2d10be98cc7e01'] ...
Start calling SSM Document deployTestSM-aem-stack-manager-main-stack-SSMCommandsStack-B4JXFJJAEMR1-WaitUntilReady-Z257ISKP9D0S ...
Finished calling SSM Document deployTestSM-aem-stack-manager-main-stack-SSMCommandsStack-B4JXFJJAEMR1-WaitUntilReady-Z257ISKP9D0S.
[blochfs/publish] Command ID: f33d41e5-11cc-4132-980b-860412cc54a8 ...
[blochfs/publish] Finished sending command COMPACT_REMAINING_PUBLISHERS / PUBLISH_READY
END RequestId: 4b4edd1e-b18b-11e8-9467-89b6b872ecee
REPORT RequestId: 4b4edd1e-b18b-11e8-9467-89b6b872ecee Duration: 2612.28 ms Billed Duration: 2700 ms Memory Size: 128 MB Max Memory Used: 72 MB
START RequestId: 021098fe-b18c-11e8-b239-3d0b7a21a89a Version: $LATEST
[blochfs] Start updating ASG blochfs-aem-full-set-main-stack-PublishDispatcherStack-X0LO9LX9R3CV-PublishDispatcherAutoScalingGroup-1W3ID11J40LGF to 0 instances ...
[blochfs] Finish updating ASG blochfs-aem-full-set-main-stack-PublishDispatcherStack-X0LO9LX9R3CV-PublishDispatcherAutoScalingGroup-1W3ID11J40LGF to 0 instances.
[blochfs] Start entering instance [u'i-05b1707408754053d'] into standby in ASG blochfs-aem-full-set-main-stack-PublishDispatcherStack-X0LO9LX9R3CV-PublishDispatcherAutoScalingGroup-1W3ID11J40LGF ...
[blochfs] Finish entering instance [u'i-05b1707408754053d'] into standby in ASG blochfs-aem-full-set-main-stack-PublishDispatcherStack-X0LO9LX9R3CV-PublishDispatcherAutoScalingGroup-1W3ID11J40LGF.
[blochfs/publish] Start sending command COMPACT_REMAINING_PUBLISHERS / STOP_PUBLISH for instance ids:
[blochfs/publish] [u'i-0f4b6f5c418e9c1c6'] ...
Start calling SSM Document deployTestSM-aem-stack-manager-main-stack-SSMCommandsStack-B4JXFJJAEMR1-ManageService-1MSVLEJ131TV5 ...
Finished calling SSM Document deployTestSM-aem-stack-manager-main-stack-SSMCommandsStack-B4JXFJJAEMR1-ManageService-1MSVLEJ131TV5.
[blochfs/publish] Command ID: 7bf64e20-4162-4cc0-91cf-69874096295e ...
[blochfs/publish] Finished sending command COMPACT_REMAINING_PUBLISHERS / STOP_PUBLISH
END RequestId: 021098fe-b18c-11e8-b239-3d0b7a21a89a
REPORT RequestId: 021098fe-b18c-11e8-b239-3d0b7a21a89a Duration: 3158.74 ms Billed Duration: 3200 ms Memory Size: 128 MB Max Memory Used: 72 MB
START RequestId: 1ccf5e15-b18c-11e8-abb6-65f1bc2d66b3 Version: $LATEST
[blochfs/publish] Start sending command COMPACT_REMAINING_PUBLISHERS / COMPACT_PUBLISH for instance ids:
[blochfs/publish] [u'i-0f4b6f5c418e9c1c6'] ...
Start calling SSM Document deployTestSM-aem-stack-manager-main-stack-SSMCommandsStack-B4JXFJJAEMR1-OfflineCompactionSnapshotFullset-HDTC3CRHREH3 ...
Finished calling SSM Document deployTestSM-aem-stack-manager-main-stack-SSMCommandsStack-B4JXFJJAEMR1-OfflineCompactionSnapshotFullset-HDTC3CRHREH3.
[blochfs/publish] Command ID: 46ab8780-b437-471e-976f-4b2c7c37dfe3 ...
[blochfs/publish] Finished sending command COMPACT_REMAINING_PUBLISHERS / COMPACT_PUBLISH
END RequestId: 1ccf5e15-b18c-11e8-abb6-65f1bc2d66b3
REPORT RequestId: 1ccf5e15-b18c-11e8-abb6-65f1bc2d66b3 Duration: 687.03 ms Billed Duration: 700 ms Memory Size: 128 MB Max Memory Used: 72 MB
START RequestId: 26bb0ad6-b18c-11e8-abec-bb07c959a4d6 Version: $LATEST
[blochfs/publish] Start sending command COMPACT_REMAINING_PUBLISHERS / START_PUBLISH for instance ids:
[blochfs/publish] [u'i-0f4b6f5c418e9c1c6'] ...
Start calling SSM Document deployTestSM-aem-stack-manager-main-stack-SSMCommandsStack-B4JXFJJAEMR1-ManageService-1MSVLEJ131TV5 ...
Finished calling SSM Document deployTestSM-aem-stack-manager-main-stack-SSMCommandsStack-B4JXFJJAEMR1-ManageService-1MSVLEJ131TV5.
[blochfs/publish] Command ID: 74a252fc-abce-4d4e-b6ae-911d1257b1e2 ...
[blochfs/publish] Finished sending command COMPACT_REMAINING_PUBLISHERS / START_PUBLISH
END RequestId: 26bb0ad6-b18c-11e8-abec-bb07c959a4d6
REPORT RequestId: 26bb0ad6-b18c-11e8-abec-bb07c959a4d6 Duration: 749.04 ms Billed Duration: 800 ms Memory Size: 128 MB Max Memory Used: 72 MB
START RequestId: 317578c6-b18c-11e8-91e4-a70ed8a86ebc Version: $LATEST
[blochfs] Start exiting instance [u'i-05b1707408754053d'] from standby in ASG blochfs-aem-full-set-main-stack-PublishDispatcherStack-X0LO9LX9R3CV-PublishDispatcherAutoScalingGroup-1W3ID11J40LGF ...
[blochfs] Finish exiting instance [u'i-05b1707408754053d'] from standby in ASG blochfs-aem-full-set-main-stack-PublishDispatcherStack-X0LO9LX9R3CV-PublishDispatcherAutoScalingGroup-1W3ID11J40LGF.
[blochfs] Start updating ASG blochfs-aem-full-set-main-stack-PublishDispatcherStack-X0LO9LX9R3CV-PublishDispatcherAutoScalingGroup-1W3ID11J40LGF to 4 instances ...
[blochfs] Finish updating ASG blochfs-aem-full-set-main-stack-PublishDispatcherStack-X0LO9LX9R3CV-PublishDispatcherAutoScalingGroup-1W3ID11J40LGF to 4 instances.
Unlock DynamoDB Table deployTestSM-AemStackManagerTable
[blochfs] Offline compaction backup for environment finished successfully
END RequestId: 317578c6-b18c-11e8-91e4-a70ed8a86ebc
REPORT RequestId: 317578c6-b18c-11e8-91e4-a70ed8a86ebc Duration: 1273.54 ms Billed Duration: 1300 ms Memory Size: 128 MB Max Memory Used: 72 MB
<https://ap-southeast-2.console.aws.amazon.com/cloudwatch/home?region=ap-southeast-2>

"This e-mail and any attachments to it (the "Communication") is, unless otherwise stated, confidential, may contain copyright material and is for the use only of the intended recipient. If you receive the Communication in error, please notify the sender immediately by return e-mail, delete the Communication and the return e-mail, and do not read, copy, retransmit or otherwise deal with it. Any views expressed in the Communication are those of the individual sender only, unless expressly stated to be those of Australia and New Zealand Banking Group Limited ABN 11 005 357 522, or any of its related entities including ANZ Bank New Zealand Limited (together "ANZ"). ANZ does not accept liability in connection with the integrity of or errors in the Communication, computer virus, data corruption, interference or delay arising from or in respect of the Communication."
```

**Comment by mbloch1986** [comment]
Released in Version 1.3.1.
